src/cmd/src/standalone.rs

```diff
@@ -93,6 +93,7 @@ impl SubCommand {
 pub struct StandaloneOptions {
     pub mode: Mode,
     pub enable_telemetry: bool,
+    pub initialize_region_in_background: bool,
```

Comment:
Is it necessary to export this as an option? Or we can set it to `true` as the default behavior meets most requirements and scenario

Reply:
I believe the Standalone mode doesn't need this option and should not initialize in the background.